src/Cardano/Wallet.purs

```diff
@@ -59,9 +64,18 @@ type Cip30DataSignature =
 -- | Wrapper for wallet names. see nami, flint, etc for examples.
 newtype WalletName = WalletName String
 
+apiVersion :: WalletName -> Effect String
```

Comment:
Use `getApiVersion`

Reply:
Yep, renamed it. The naming convention was quite obvious actually. But I didn't see it nonetheless.